sdk/core/core-lro/README.md

```diff
@@ -1,18 +1,24 @@
 # Azure Core LRO client library for JavaScript
 
-Azure's Core LRO is a JavaScript library that provides an API that aims to
-allow the azure-sdk-for-js public libraries to implement fully featured pollers
-to manage Long Running Operations (LROs) with the Azure services.
+`@azure/core-lro` is a JavaScript library that allows developers to build pollers that keep track of Long Running Operations (LROs) that require consecutive calls to endpoints provided by the Azure services to update a local representation of the status of the remote operation until its completion.
 
-@azure/core-lro is made following our guidelines here: https://azure.github.io/azure-sdk/typescript_design.html#ts-lro
+Even though this library is publicly available for installation, this library is not intended to be used outside of the projects inside of the `azure-sdk-for-js` repository.
```

Comment:
you could say "this library is intended to be used as a dependency of official Azure SDK packages and not intended for inclusion in your projects directly."

Reply:
You may also want to start the sentence **Please note:**

Reply:
Thank you! I'll do this in a bit